network-api/networkapi/settings.py

```diff
@@ -249,7 +249,6 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
 
-    'wagtail.core.middleware.SiteMiddleware',
```

Comment:
as per Wagtail 2.9, see https://docs.wagtail.io/en/stable/releases/2.9.html#sitemiddleware-and-request-site-deprecated

Reply:
Yep, good catch!